docs/formats.rst

```diff
@@ -231,36 +231,36 @@ is one of many standards in this area.
 
 XLIFF is usually used as bilingual, but Weblate supports it as monolingual as well.
```

Comment:
There is no link to the XLIFF specification.
other than maybe "attribute as defined in the XLIFF standard, see :ref:`xliff-flags`." earlier.

It should IMO be similar to:
```
.. seealso::

    `Specification of Joomla language files <https://docs.joomla.org/Specification_of_language_files>`_,
```

Reply:
Good point, the specification is here: http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html

Reply:
link added